chapters/data-analysis.tex

```diff
@@ -1,292 +1,487 @@
+and where different files can be found in the project folder. 
+That is, it should contain all the information needed to interact with a project's data work.
+
+% Version control
+Finally, everything that can be version-controlled should be. 
+Version control allows you to effectively track code edits,
+including the addition and deletion of files. 
+This way you can delete code you no longer need, 
+and still recover it easily if you ever need to get back previous work.
+Both analysis results and data sets will change with the code.
+You should have each of them stored with the code that created it.
+If you are writing code in Git/GitHub,
+you can output plain text files such as \texttt{.tex} tables
+and metadata saved in \texttt{.txt} or \texttt{.csv} to that directory.
+Binary files that compile the tables,
+as well as the complete data sets, on the other hand,
+should be stored in your team's shared folder. 
+Whenever data cleaning or data construction codes are edited,
+use the master script to run all the code for your project.
+Git will highlight the changes that were in data sets and results that they entail. 
+
+%------------------------------------------------
+
+\section{Data cleaning}
+
+% intro: what is data cleaning -------------------------------------------------
+Data cleaning is the first stage of transforming the data you received from the field into data that you can analyze.\sidenote{\url{https://dimewiki.worldbank.org/wiki/Data\_Cleaning}}
+The cleaning process involves (1) making the data set easily usable and understandable, 
+and (2) documenting individual data points and patterns that may bias the analysis.
+The underlying data structure does not change.
+The cleaned data set should contain only the variables collected in the field.
+No modifications to data points are made at this stage, except for corrections of mistaken entries.
+
+Cleaning is probably the most time consuming of the stages discussed in this chapter.
+This is the time when you obtain an extensive understanding of  the contents and structure of the data that was collected.
+Explore your data set using tabulations, summaries, and descriptive plots.
+You should use this time to understand the types of responses collected, both within each survey question and across respondents.
+Knowing your data set well will make it possible to do analysis.
+
+
+% Deidentification ------------------------------------------------------------------
+The initial input for data cleaning is the raw data.
+It should contain only materials that are received directly from the field.
+They will invariably come in a host of file formats and nearly always contain personally-identifying information.\index{personally-identifying information}
+These files should be retained in the raw data folder \textit{exactly as they were received}.
+The folder must be encrypted if it is shared in an insecure fashion,\sidenote{\url{https://dimewiki.worldbank.org/wiki/Encryption}}
```

Comment:
All secure sharing includes encryption. So I think we should say that this folders must always be encrypted. And we can add that they need to make sure the software they use for sharing does indeed encrypt

Reply:
I'm not sure I get it. In our case, for example, if a file is in one drive, we don't need to encrypt it, right? Is it because one drive already encrypts it?

Reply:
I understand there are two ways encryption can happen.

Either the whole storage is encrypted at rest and permissioned/passworded/2FA'd (Bank-managed laptop; Bank-managed OneDrive), in which case the file does not need to be separately encrypted from the system; or

The storage is not that type (personal machine or personal storage like Dropbox), in which case the file itself does needs to be separately encrypted.

Do we broadly agree? For almost all practical applications this is going to be the second case...

Reply:
@luizaandrade , yes the only reason we do not need to encrypt it is that the WB OneDrive system does encrypt it appropriately for us.

@bbdaniels , yes, but there is more detail. DropBox is encrypted at rest but they own the key, and since DropBoc Inc is not on the IRB that is an issue. 

I think where I am sub-consciously taking this argument is that while encryption is binary (a file is either encrypted or not) a whole system is much more complex. If you share with DropBox the file is encrypted in transit and at rest, but the way it is encrypted at rest is not satisfactory.

Expressing this in a correct but still in a way helpful to the reader will take space in the book. So I guess the question is if we want to take up space for that.

Reply:
97b2ccd reads:

>Additionally, no one who is not listed in the IRB should be able to access its content, not even the company providing file-sharing services. Check if your organization has guidelines on how to store data securely, as they may offer an institutional solution. If that is not the case, you will need to encrypt the file and make sure that only IRB-listed team members have the encryption key.\sidenote{\url{https://dimewiki.worldbank.org/wiki/Encryption}}.

Reply:
I really like this. I just made a small edit in be5e8f020c44e4579594fff706d2c0d626f2ad7b. We seem to be working on this chapter at the same time, so I created branch for small edits like this that I will merge to `la-ch6-from-dev` as soon as I have read through this